erdpy/wallet/pem.py

```diff
@@ -25,6 +25,26 @@ def parse(pem_file):
     return seed, pubkey
 
 
+def parse_validator_pem(pem_file):
```

Comment:
is there a need for two functions to process a pem file? 
I think the only difference is the keys length, so maybe if you give as well the seed/sk length as parameter to parse function, and use that when splitting between the seed and pubkey, you can use "parse()" for both? 
Or use this second function for both? 

Just a thought

Reply:
Good suggestion. I added a ticket for this task because we also have to do some refactoring there.
https://elrondnetwork.atlassian.net/browse/EN-7135.